gallery_dl/extractor/reactor.py

```diff
@@ -16,7 +16,7 @@
 import json
 
 
-BASE_PATTERN = r"(?:https?://)?([^/.]+\.reactor\.cc)"
+BASE_PATTERN = r"(?:https?://)?(([^/.]+\.)?reactor\.cc)"
```

Comment:
```suggestion
BASE_PATTERN = r"(?:https?://)?((?:[^/.]+\.)?reactor\.cc)"
```
Don't create a new capture group.
This would shift the original group 2 to index 3, and most extractors rely on the contents of group 2 for metadata:
https://github.com/mikf/gallery-dl/blob/8789f5b81d812b47f5be420a2246cd2570b8e56e/gallery_dl/extractor/reactor.py#L158